### PR TITLE
docs: updated contributing and encrypted images pages

### PR DESCRIPTION
**Pull request by pawelpros**

### CODE REVIEW

File: content/en/docs/features/encrypted-images.md [hunk 1]
```diff
@@ -96,12 +104,87 @@ In this example we default to the Cloud API Adaptor runtime, adjust this dependi
...
+
+{{% tab header="Bare metal" %}}
+
+```yaml
+io.katacontainers.config.hypervisor.kernel_params: "agent.guest_components_rest_api=all agent.aa_kbc_params=cc_kbc::${KBS_ADDRESS}"
+```
+
+<br/>
+
+**Note:** The KBS address parameter `${KBS_ADDRESS}` should be in format `scheme://host:port` where scheme://host:port represents the address.
```
Comment (JakubLedworowski) [inline content/en/docs/features/encrypted-images.md:135]:
```suggestion
**Note:** The KBS address parameter `${KBS_ADDRESS}` should be in format `scheme://host:port`
```

File: content/en/docs/features/encrypted-images.md [hunk 2]
```diff
@@ -96,12 +104,87 @@ In this example we default to the Cloud API Adaptor runtime, adjust this dependi
...
+   
+   [kbc]
+   name = 'cc_kbc'
+   url = '${KBS_ADDRESS}'
+   '''
+   EOF
+   ```
+3. Run command and use base64 encoded output as input for `${INIT_DATA_B64}` variable:
+   ```bash
+   cat $HOME/initdata.toml | base64 -w0
```
Comment (JakubLedworowski) [inline content/en/docs/features/encrypted-images.md:179]:
```suggestion
   INIT_DATA_B64=$(cat $HOME/initdata.toml | base64 -w0)
```

File: content/en/docs/features/encrypted-images.md [hunk 3]
```diff
@@ -74,7 +81,8 @@ skopeo inspect dir:./oci/output | jq '.LayersData[0].Annotations["org.opencontai
 }
 ```
 
-Finally the resulting encrypted image can be provisioned to an image registry.
+Finally, the resulting encrypted image can be provisioned to an image registry. <br>
+Authenticated registries are also described [here](./authenticated-registries.md).
```
[the remark this answers is not in the document]
Comment (pawelpros) [reply content/en/docs/features/encrypted-images.md:85]: Removing

**Review by JakubLedworowski — APPROVED** [review]

**Review by JakubLedworowski — APPROVED** [review]

File: content/en/docs/features/encrypted-images.md [hunk 4]
```diff
@@ -120,19 +149,93 @@ spec:
       labels:
         app: nginx
       annotations:
+        io.katacontainers.config.hypervisor.kernel_params: "agent.guest_components_rest_api=all agent.aa_kbc_params=cc_kbc::${KBS_ADDRESS}"
```
Comment (fitzthum) [inline content/en/docs/features/encrypted-images.md:152]: The `rest_api` settings are mostly orthogonal to encrypted images. Maybe we should leave that out. Also setting it to all is not secure by default.

File: content/en/docs/features/encrypted-images.md [hunk 5]
```diff
@@ -86,22 +93,44 @@ skopeo copy dir:./oci/output "docker://${ENCRYPTED_IMAGE}"
 Prior to launching a Pod the image key needs to be provisioned to the Key Broker's repository. For a KBS deployment on Kubernetes using the local filesystem as repository storage it would work like this:
 
 ```bash
-kubectl exec deploy/kbs -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
-cat "$KEY_FILE" | kubectl exec -i deploy/kbs -- tee "/opt/confidential-containers/kbs/repository/${KEY_PATH}" > /dev/null
+kubectl exec deploy/kbs -n coco-tenant -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
```
Comment (fitzthum) [inline content/en/docs/features/encrypted-images.md:96]: Where does the coco-tenant runtime class come from?

**Review by fitzthum — COMMENTED** [review]
Thanks for the updates. Two small comments

File: content/en/docs/features/encrypted-images.md (hunk 5 above)
Comment (pawelpros) [reply content/en/docs/features/encrypted-images.md:96]: It's not runtime class but default namespace which is used for KBS (coco-tenant) - in documentation (I checked couple other pages) also it's used with namespace and as far as I know KBS is never deployed with default namespace

File: content/en/docs/features/encrypted-images.md (hunk 4 above)
Comment (pawelpros) [reply content/en/docs/features/encrypted-images.md:152]: OK - I'll remved part of param and leave only cc_kbs related part

File: content/en/docs/features/encrypted-images.md (hunk 5 above)
Comment (fitzthum) [reply content/en/docs/features/encrypted-images.md:96]: This namespace is in the yamls provided in the Trustee repo, but it's not in the trustee-operator repo. Do we still use it when installing via the operator? We can probably merge this either way, but we may have an inconsistency here.

Comment (PierreLeGuen) [reply content/en/docs/features/encrypted-images.md:96]: This `coco-tenant` namespace is causing issues when interacting with kbs.

```
$ kubectl exec deploy/kbs -n coco-tenant -- mkdir -p "/opt/confidential-containers/kbs/repository/$(dirname "$KEY_PATH")"
Error from server (NotFound): namespaces "coco-tenant" not found
```

Comment (fitzthum) [reply content/en/docs/features/encrypted-images.md:96]: Ok, we've yet to provide good instructions for setting up attestation in the docs. I picture offloading the specifics of deploying and provisioning the KBS to other pages where we could address this sort of thing. For now, maybe make a note idk.

**Review by fitzthum — APPROVED** [review]
LGTM

We've maybe discovered a little issue with the runtime class but that's not directly related to this PR

File: content/en/docs/features/encrypted-images.md (hunk 5 above)
Comment (pawelpros) [reply content/en/docs/features/encrypted-images.md:96]: I'll remove coco-tenant and replace it with note about it

**Pull request merged**